### ISSUE TITLE
Products with options / combination should have text for selected option

### ISSUE THREAD

**Issue by 01park** [PrestaShop/PrestaShop#21006]
#### Is your feature request related to a problem?

Yes

#### Describe the solution you'd like

I would like it to be more visible to the customer the option they select when chosen a combination. 

For example, when one selects the color yellow, instead of just highlighting the color yellow, have above or beneath the option yellow in text. This could make things more clear to the customer, for example ones that are color blind.

**Comment by sarahdib** [comment]
@PrestaShop/product-team could you check this improvement ?

Thank you :)

**Comment by LouiseBonnard** [comment]
I guess it's a question for the @PrestaShop/ux-team. ;-)

**Comment by prestascott** [comment]
Hello everyone!

@01park Thank you for your report. :) 

Can you provide us some screenshots so we can exactly see what you mean? Thanks.

**Comment by 01park** [comment]
<img width="1067" alt="Screenshot 2020-09-16 at 16 54 09" src="https://user-images.githubusercontent.com/46492443/93362240-ab5e5100-f83d-11ea-8655-3f148e8c6608.png">

Hi, 

This is  an example.

Behind the combination word, in this case color, put the option the customer selects.

If it was size, then size: S or size: XL

It's to make things more clear.

**Comment by prestascott** [comment]
Thank you, it's clearer to me. As a colorblind person, I agree that this will really be helpful and provide our users a better accessibility. 

Also, we can add an hover on each combination to show the text. Thanks to @LouiseBonnard for submitting this idea!

I will add this issue to the Epic. :)

**Comment by 01park** [comment]
Hi,

Thanks, I hope it will be applied in the future

**Comment by LouiseBonnard** [comment]
What about turning it into a parameter in the _Shop Parameters > Product Settings > Product page_ section of the back office? @Julievrz, any idea about the wording?

**Comment by Julievrz** [comment]
What about : 

"Display a text label to describe the selected product combination (e.g., selected color: black)" 

@LouiseBonnard What do you think?

Do we add another wording for the hover effect? Something like...

"Display selected product combination's information on hover (e.g., color: black)"

**Comment by LouiseBonnard** [comment]
After a small research, I have found that it is called a ‘mouseover text’ but I don't know if it will be understandable enough for the users, it sounds like a technical vocabulary. Anyway, I really like your wording suggestions!

I'd go for _Display a text label to describe the selected attribute_ as the feature label (in _Admin.Shopparameters.Feature_) and add the example in the tooltip (in _Admin.Shopparameters.Help_). We don't write examples in the feature label.

**Comment by NeOMakinG** [comment]
It's not a mouseover text, this is already the case, when you hover an element you got the value inside a title attribute at the moment.

It's about static data after a click :

![image](https://user-images.githubusercontent.com/14963751/97164737-ddd76400-1782-11eb-85d2-9adb2cd57ee6.png)

Is this enough ? Providing a PR for this

**Comment by Julievrz** [comment]
@NeOMakinG : If I understand well, the idea is to have both (a mouseover text and static data after a click). I am maybe wrong but I tried to pass my mouse over the black/white square and no text appeared. That is why I proposed 2 wordings.

**Comment by NeOMakinG** [comment]
As it's a title attribute, you should wait a bit for it to be displayed, this is an accessibility attribute

**Comment by Julievrz** [comment]
Mhm, that's weird. It does not work on my side.

**Comment by NeOMakinG** [comment]
<img width="111" alt="Screenshot 2020-10-27 at 11 08 19" src="https://user-images.githubusercontent.com/14963751/97287493-f1e39a00-1844-11eb-8f9b-55dd3964b228.png">

And the title attribute is supported since years ^^

**Comment by Julievrz** [comment]
> <img alt="Screenshot 2020-10-27 at 11 08 19" width="111" src="https://user-images.githubusercontent.com/14963751/97287493-f1e39a00-1844-11eb-8f9b-55dd3964b228.png">
> 
> And the title attribute is supported since years ^^

Hello @NeOMakinG 

I saw that the PR was just merged. I just wanted to know if we should do something specific to make the mouseover appear. Indeed, with @LouiseBonnard when we tried to pass the mouse over the black/white square on the product page, no text appears (even after waiting for a bit). However, we do see it on the products listing page. It happens that our shop is not up to date compared to the developer's one so maybe that is the reason...
